tutorial/notebooks/images_are_arrays.md

```diff
@@ -0,0 +1,2 @@
+(numpy-arrays-tutorial)=
+# Part 1: Images are just NumPy arrays
```

Comment:
Probably we still want these files to have 0, 1, 2... prefixes because although the TOC can sort them, the jupyter notebook / lab interface will not.